client/src/components/Form/FormField.jsx

```diff
@@ -0,0 +1,22 @@
+import { Field, ErrorMessage } from 'formik';
+import { TextError } from './TextError';
+const FormField = ({ placeholder, name, ...rest }) => {
+  return (
+    <div className='form-group'>
+      <Field
+        id={name}
+        name={name}
+        {...rest}
+        placeholder={placeholder}
+        className='form-control  mb-3 w-50 text-white '
```

Comment:
Pass the className as a props.
```suggestion
        className={`form-control  ${className} `}
```